src/UI/class-settings-page.php

```diff
@@ -19,6 +19,20 @@
  * Renders the wp-admin Parse.ly plugin settings page.
  *
  * @since 3.0.0
+ *
+ * @phpstan-import-type ParselyOptions from Parsely
+ *
+ * @phpstan-type SettingArguments array{
+ *   option_key: string,
+ *   label_for: string,
+ *   title?: string,
+ *   help_text?: string,
+ *   yes_text?: string,
+ *   filter?: string,
+ *   optional_args?: array<string, string>,
+ *   select_options?: array<string, string>,
+ *   radio_options?: array<string, string>,
+ * }
```

Comment:
Define setting arguments that we need to print controls of each setting. Here optional fields are marked as `?:` and these will also need null check below.